src/lhttpc_manager.erl

```diff
@@ -271,7 +271,7 @@ init(Options) ->
         true ->
             % Make sure that the ssl random number generator is seeded
             % This was new in R13 (ssl-3.10.1 in R13B vs. ssl-3.10.0 in R12B-5)
-            apply(ssl, seed, [crypto:rand_bytes(255)]);
+            apply(ssl, seed, [crypto:strong_rand_bytes(255)]);
```

Comment:
Have we? [This](https://github.com/erlang/otp/blob/OTP-19.3.4/lib/crypto/c_src/crypto.c#L2037) is `rand_bytes` behind the scenes as of OTP 19 - it uses OpenSSL's [RAND_pseudo_bytes](https://www.openssl.org/docs/man1.1.0/crypto/RAND_bytes.html), which looks like it doesn't fail on lack of entropy. Mmmh..